modules/wri_admin/wri_admin.module

```diff
@@ -99,3 +105,114 @@ function wri_admin_theme_suggestions_entity_moderation_form(array $variables) {
 
   return $suggestions;
 }
+
+/**
+ * Validation function to ensure the media name is not just numbers.
+ */
+
+/**
+ * Validation function to ensure the media name is not just numbers.
+ */
+function wri_admin_validate_media_name(array &$form, FormStateInterface $form_state) {
+  \Drupal::logger('wri_admin')->notice('Validation triggered for media name.');
+
+  // Log available form state keys (safe debugging).
+  $form_state_keys = array_keys($form_state->getValues());
+  \Drupal::logger('wri_admin')->notice('Form state keys: @keys', ['@keys' => implode(', ', $form_state_keys)]);
+
+  // Check if the 'name' field exists in the form state.
+  $name_values = $form_state->getValue('name');
```

Comment:
It looks like if you upload the file from a node page, the value is located within "Media" -- Try uploading a file named '1.jpg' as the hero image on an article.